OpenOversight/app/models.py

```diff
@@ -32,10 +32,18 @@ class Department(db.Model):
     id = db.Column(db.Integer, primary_key=True)
     name = db.Column(db.String(255), index=True, unique=True, nullable=False)
     short_name = db.Column(db.String(100), unique=False, nullable=False)
+    uii = db.Column(db.String(100), unique=False, nullable=True)
```

Comment:
it looks like this field is to denote that the department uses a unique internal ID in addition to badge number - how would this be set / updated? if the previous statmeent is correct, should it be a Boolean? 

Nit: can we name this something more expressive/explicit e.g. `is_using_unique_internal_id`

Reply:
The field `uii` contains the exact term that the department uses for the unique internal id, for example, for NYPD this would be the tax registry number.  I created this field in order to display the appropriate term in step 2 of the Find Officer Form.  (At first I just asked for the "unique internal identifier," but I worried that would be unclear to users.)
I don't think the field should be a Boolean.  Do you want me to leave the field as it is, named `uii`, or do you still want something more explicit?  Perhaps `uii_term`, `unique_internal_identifier_label`, or `uii_phrase`?  I'm fine with leaving it as `uii`, but let me know what you think.

Reply:
IMO keeping the field as is and renaming it to `unique_internal_identifier_label` makes the most sense. No need for a separate boolean field when checking for a null value in the existing column would work just as well.